server/storage/mvcc/kvstore_compaction.go

```diff
@@ -73,7 +74,7 @@ func (s *store) scheduleCompaction(compactMainRev int64, keep map[revision]struc
 		select {
 		case <-time.After(batchInterval):
 		case <-s.stopc:
-			return false
+			return fmt.Errorf("stopped")
```

Comment:
"compaction interrupted due to store stop signal" ?

Reply:
Done